datascience/util.py

```diff
@@ -22,3 +24,30 @@ def percentile(p, arr=None):
     if arr is None:
         return lambda arr: percentile(p, arr)
     return np.percentile(arr, p, interpolation='higher')
+
+def plotCDFarea(rbound, lbound=None, mean=0, sd=1):
+    """Plots a normal curve with specified parameters and area below curve shaded
+    between ``lbound`` and ``rbound``.
+
+    Args:
+        ``rbound`` (numeric): right boundary of shaded region
+
+        ``lbound`` (numeric): left boundary of shaded region; by default is negative infinity
+
+        ``mean`` (numeric): mean/expectation of normal distribution
+
+        ``sd`` (numeric): standard deviation of normal distribution
+    """
+    llabel = lbound
+    if lbound is None:
+        lbound = sd * -3.5 + mean
```

Comment:
This `3.5` is basically just a constant used to denote infinity, right? If so, let's define a constant `INF = 3.5 * sd` and use `-INF` and `INF` throughout this method instead of [magic numbers](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad).